### PR TITLE
Update tutor.txt

### PR DESCRIPTION
**Pull request by liigo**
return to normal mode before enter command mode

### CODE REVIEW

**Comment by archseer** [comment]
\cc @Omnikar

File: runtime/tutor.txt [hunk 1]
```diff
@@ -140,8 +140,9 @@ _________________________________________________________________
     terminal.
  2. Open a file in Helix by running: hx FILENAME
  3. Make some edits to the file.
- 4. Press the : key to enter command mode.
- 5. Type w or write, and press <ENTER> to save the file.
+ 4. Press <ESC> to enter Normal mode.
```
Comment (pickfire) [inline runtime/tutor.txt:143]: I don't see why this is necessary given that we teach them how to go back to normal mode. Maybe we can just say `In Normal mode, press the : ...`?

Comment (Omnikar) [reply runtime/tutor.txt:143]: Yeah, and by this point it's already been taught that keys are typed in insert mode, and do special actions in normal mode. Additionally, "make some edits" doesn't necessarily mean insertion; they can also delete text as has been taught by this point.

**Comment by pickfire** [comment]
@liigo Thanks for submitting the pull request but as @Omnikar mentioned, by that point it is cleared that user already knows how to use command mode, if any the change should be done in other parts and not here to make it clear.

Closing this for now but feel free to reopen if you have better reason.

**Pull request closed without merging**